### PR TITLE
packages: busybox: enable rfkill applet

### PR DESCRIPTION
**Pull request by mklein-de**

### CODE REVIEW

**Comment by lrusak** [comment]
It would probably be helpful to add a description about _why_ you want this package added :smile_cat: 

Also, this package isn't a dependent so it won't be installed by default, is that intended?

File: packages/debug/rfkill/package.mk [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (chewitt) [inline packages/debug/rfkill/package.mk:1]: We prefer `GPL-2.0` unless there is a deliberate reason for using `GPL-2.0-or-later`

**Comment by chewitt** [comment]
Perhaps include this with the "network tools" add-on bundle?

**Comment by HiassofT** [comment]
busybox contains an rfkill applet https://git.busybox.net/busybox/tree/miscutils/rfkill.c?h=1_18_stable - maybe just enable that?

**Comment by mklein-de** [comment]
> busybox contains an rfkill applet https://git.busybox.net/busybox/tree/miscutils/rfkill.c?h=1_18_stable - maybe just enable that?

yup, that does it ;-)

File: packages/debug/rfkill/package.mk (hunk 1 above)
Comment (mklein-de) [reply packages/debug/rfkill/package.mk]: no reason for this, just copy&paste

**Review by HiassofT — APPROVED** [review]

**Pull request merged**